documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,183 @@
+... (Optional) Select the `Add project(s) to working set` check box to add additional projects to the working set.
+... (Optional) Click `Advanced` to view additional optional advanced configuration for the new Maven project, such as whether to `Resolve Workspace projects`, `Profiles`, and any `Name templates` to set a desired naming convention for the project.
++
+.Create a New Maven Project
+image::mvnbasic_project_name_location.png[Create a New Maven Project]
++
+.. When the configuration is complete, click `Next` to continue.
+. In the second step of the `New Maven Project` wizard, enter the archetype details:
+.. Ensure that the `Show the last version of Archetype only` check box is selected. This ensures that only the latest version of each archetype displays.
+.. Select an archetype based on the purpose of the project you are creating. Use the keyword `maven-archetype-quickstart` in the `Filter` field to locate a sample Maven project archetype.
++
+.Select a Project Archetype
+image::mvnbasic_select_archetype.png[Select a Project Archetype]
++
+.. Click `Next` to continue.
+. In the third step of the `New Maven Project` wizard, specify the archetype parameters:
+.. Enter the desired group ID in the `Group Id` field.
+.. Enter the desired artifact ID in the `Artifact Id` field.
+.. Set the `Version` field’s value to `0.0.1-SNAPSHOT` or a similar value. For details about the appropriate version build numbers, see http://books.sonatype.com/mvnref-book/reference/pom-relationships-sect-pom-syntax.html[Project Versions]
+.. The `Package` field automatically populates based on the entered group and artifact IDs.
++
+.Specify Project Archetype Parameters
+image::mvnbasic_specify_group_parameters.png[Specify Project Archetype Parameters]
++
+.. Click `Finish` to conclude the new Maven project creation wizard.
+
+*Result:* Your new Maven project is created and appears in the `Project Explorer` view.
+
+[[New_Module]]
+== 2. Create a New Maven Module
+
+Each Maven project can include multiple Maven modules. Follow the instructions to create your first Maven module:
+
+=== Prerequisites
+. You must have an existing Maven project available to use for the new module. See <<New_Project,Create a New Maven Project>> for instructions to create a new Maven project.
+. Set the `Packaging` option to pom as follows:
+.. In the `Project Explorer` view, expand your existing Maven project and double-click `pom.xml` in the expanded view.
+.. The `Overview` tab appears with configuration information for your Maven project.
+.. In the `Packaging` drop-down menu, select the `pom` option.
++
+.Set the Project's Packaging Option
+image::mvnbasic_set_packaging.png[Set the Project's Packaging Option]
++
+.. Click the save icon to save the changes.
+
+=== Creating a New Maven Module
+. Create a new Maven module as follows:
+.. In your workspace, navigate to menu:File[New > Other].
```

Comment:
@fbricon, could you elaborate in this please? I assumed since we have separate options for "Maven Project" and "Maven Module" under Other, these are the two used for a new Maven project and module respectively. Would project instead here just create another project rather than a module?